<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 5px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>MULTIPLEXER</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="select1CheckBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sel 1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="select2CheckBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sel 2</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>16</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>16</width>
      <height>91</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Button</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>select1CheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select2CheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleSelectLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectOutput()</slot>
  <slot>toggleSelectLines()</slot>
 </slots>
</ui>
